dts/arm/cypress/psoc6_cm0.dtsi

```diff
@@ -0,0 +1,28 @@
+#include <arm/armv6-m.dtsi>
+#include <cypress/psoc6.dtsi>
+
+/ {
+	cpus {
+		cpu@0 {
```

Comment:
Why repeat here what's in included psoc6.dtsi?

Reply:
Made by analogy to dts\arm\nxp\nxp_lpc54xxx.dtsi, nxp_lpc54xxx_m0.dtsi, nxp_lpc54xxx_m4.dtsi